zerver/lib/avatar.py

```diff
@@ -43,7 +43,7 @@ def get_avatar_url(avatar_source, email, avatar_version, medium=False):
     url += '&version=%d' % (avatar_version,)
     return url
 
-def _get_unversioned_avatar_url(avatar_source, email=None, realm_id=None,
+def _get_unversioned_avatar_url(avatar_source, email, realm_id=None,
```

Comment:
You should change the callers to pass it positionally as well.  As a style matter, we avoid mixing kwargs and positionally args, since code that does that is usually hard to read.  With Python 3, it'll be possible to enforce this in the codebase, but you can't do it in Python 2 yet.